src/macEditingCommands.ts

```diff
@@ -17,7 +17,6 @@
 
 export const macEditingCommands: {[key: string]: string|string[]} = {
   'Backspace': 'deleteBackward:',
-  'Tab': 'insertTab:',
```

Comment:
This gets filtered out somewhere in the browser because Tab always moves focus instead of creating a tab character. WebKit ignores these commands and they confuse Chromium, so I removed them.